weaviate/gql/get.py

```diff
@@ -18,6 +19,30 @@
 from weaviate.util import image_encoder_b64, _capitalize_first_letter
 
 
+@dataclass
+class BM25:
+    query: str
+    properties: Optional[List[str]]
+
+    def __str__(self) -> str:
+        ret = f'query: "{self.query}"'
+        if self.properties is not None and len(self.properties) > 0:
+            props = '","'.join(self.properties)
+            ret += f', properties: ["{props}"]'
+        return "bm25:{" + ret + "}"
+
+
+@dataclass
+class Hybrid:
+    query: str
+    alpha: float
+    vector: List[float]
```

Comment:
in other clients `alpha` and `vector` seems to be optional, so it may not be present in final hybrid string

@donomii could you clarify which params are mandatory / optional?

Reply:
alpha and vector are optional, query is not

Reply:
what happens if alpha and or vector is None?

Reply:
If alpha is none, the default will be used (currently hard-coded to 0.75).  If vector is none, then weaviate will use the configured text-to-vector module to create a vector from the "query" field, and use that